internal/server/providers/_testdata/test-server-sec.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
This looks scary, but its just a dummy private key I made specifically for the test server. It must sign test ID tokens and use HTTPS.